virttest/qemu_vm.py

```diff
@@ -1717,14 +1717,14 @@ def add_secure_guest_object(params):
         # Add device virtio-iommu, it must be added before any virtio devices
         if (params.get_boolean('virtio_iommu') and
                 devices.has_device('virtio-iommu-pci')):
-            iommu_params = {"pcie_direct_plug":
-                            params.get("virtio_iommu_direct_plug", "yes")}
+            iommu_params = {"pcie_direct_plug": "yes"}
```

Comment:
Hi @PaulYuuu 
Could you tell me from which version it starts to restrict this behavior?
Since I am afraid the upstreams would like to keep it as configured by their's requirements.
Thanks.

Reply:
virtio-iommu only supports insert to the root bus, even it can insert to other buses, but that's a wrong usage, the latest qemu already fixed it so that we can remove it safely.